<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Weapons">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Weapons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Shoot">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SHOOT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Player">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ammo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ammo:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HP: </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reload">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RELOAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Money">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Money: 0</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Equipt">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>520</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipped:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="enemy">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Slime/Без имени-1.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="atk_Bar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="ammoBar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>170</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="hpBar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>170</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="hpEnemmyBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>170</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <zorder>enemy</zorder>
   <zorder>pushButton_Weapons</zorder>
   <zorder>pushButton_Shoot</zorder>
   <zorder>pushButton_Player</zorder>
   <zorder>label_Ammo</zorder>
   <zorder>label_HP</zorder>
   <zorder>pushButton_Reload</zorder>
   <zorder>label_Money</zorder>
   <zorder>label_Equipt</zorder>
   <zorder>atk_Bar</zorder>
   <zorder>ammoBar</zorder>
   <zorder>hpBar</zorder>
   <zorder>hpEnemmyBar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEXIT">
    <property name="title">
     <string>EXIT</string>
    </property>
   </widget>
   <addaction name="menuEXIT"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
